template.config.env

```diff
@@ -0,0 +1,23 @@
+# Megabot
+BOT_TOKEN=
+
+# MongoDB
+DB_CONN_STRING=
+DB_NAME=
+
+# Budgett
+BUDGETT_TOKEN=
+# Prod
+# BUDGETT_CHAT_ID=391063932535701505
+# Staging
+BUDGETT_CHAT_ID=352255685183930369
+
+# MemeNarc
+# Prod
+# MEMENARC_CHANNEL_ID=699005722356088922
+# Staging
+MEMENARC_CHANNEL_ID=724032050717392897
+
+# Screechbot
+SCREECHBOT_LOOPDELAY=100
+SCREECHBOT_GREETING_DELAY=2000
```

Comment:
.env files are not just for secrets, but for environment specific configurations. Depending on where you deploy you may want to modify these values and it is poor practice to have environment specific configuration embedded in code files, hence why I moved these to the .env files. 

A big reason for this is ease of deployment though docker.

Reply:
That's what our env files are for. Up until now we have been using our .env files for secrets only because those are not being pushed as a part of the repo.